tests/rustdoc/line-breaks.rs

```diff
@@ -1,26 +1,37 @@
 #![crate_name = "foo"]
 
-use std::ops::Add;
 use std::fmt::Display;
+use std::ops::Add;
 
-//@count foo/fn.function_with_a_really_long_name.html //pre/br 2
-pub fn function_with_a_really_long_name(parameter_one: i32,
-                                        parameter_two: i32)
-                                        -> Option<i32> {
+// @matches foo/fn.function_with_a_really_long_name.html '//*[@class="rust item-decl"]//code' "\
+//     function_with_a_really_long_name\(\n\
+//    \    parameter_one: i32,\n\
+//    \    parameter_two: i32\n\
+//    \) -> Option<i32>$"
+pub fn function_with_a_really_long_name(parameter_one: i32, parameter_two: i32) -> Option<i32> {
     Some(parameter_one + parameter_two)
 }
 
-//@count foo/fn.short_name.html //pre/br 0
-pub fn short_name(param: i32) -> i32 { param + 1 }
+// @matches foo/fn.short_name.html '//*[@class="rust item-decl"]//code' \
+//     "short_name\(param: i32\) -> i32$"
+pub fn short_name(param: i32) -> i32 {
+    param + 1
+}
 
-//@count foo/fn.where_clause.html //pre/br 4
-pub fn where_clause<T, U>(param_one: T,
-                          param_two: U)
-    where T: Add<U> + Display + Copy,
-          U: Add<T> + Display + Copy,
-          T::Output: Display + Add<U::Output> + Copy,
-          <T::Output as Add<U::Output>>::Output: Display,
-          U::Output: Display + Copy
+// @matches foo/fn.where_clause.html '//*[@class="rust item-decl"]//code' "\
+//     where_clause<T, U>\(param_one: T, param_two: U\)where\n\
+//    \    T: Add<U> \+ Display \+ Copy,\n\
```

Comment:
The `\` right before the LF makes `htmldocck` dedent the following line, hence the “floating” `\` in this line: It stops the dedentation and “bakes” 4 literal spaces into the string literal.